src/app/components/Topbar/index.js

```diff
@@ -1,17 +1,32 @@
-import React from 'react';
-import { UTLabel } from '@widergy/energy-ui';
+import React, { useCallback } from 'react';
+import { UTLabel, UTButton } from '@widergy/energy-ui';
+import { connect } from 'react-redux';
+import { push } from 'connected-react-router';
+import { useLocation } from 'react-router';
 
 import logo from 'app/assets/logoBlanco.png';
+import { HOME } from 'constants/routes';
 
 import styles from './styles.module.scss';
 
-const Topbar = () => (
-  <div className={styles.container}>
-    <UTLabel bold large white>
-      Abloc de Notas
-    </UTLabel>
-    <img alt="logo" src={logo} className={styles.logo} />
-  </div>
-);
+const Topbar = ({ dispatch }) => {
+  const goToHome = useCallback(() => dispatch(push(HOME), [dispatch]));
 
-export default Topbar;
+  const pathName = useLocation().pathname;
+
+  return (
+    <div className={styles.container}>
+      <UTLabel bold large white>
+        Abloc de Notas
+      </UTLabel>
+      {pathName !== '/' && pathName !== '/home' && (
```

Comment:
```suggestion
      {!['/','/home'].includes(pathname) && (
```
is more scalable. those routes should be in a constants file, usually located at `src/constants/routes.js` in our projects